google/ggrc-core
# GGRC-6375 Replace can.Deferred with $.Deferred

# Issue description

CanJS 3 does not support can.Deferred.

# Steps to test the changes

General regressions, make sure async operations work fine.
`run_karma` to make sure tests stay valid.

# Solution description

Replace `can.Deferred` with `$.Deferred`.
Fix some tests that involve spies for `$.Deferred`.

# Sanity checklist




# PR Review checklist
